### DISCUSSION TITLE
Coding with an AI pair programmer [AI2780S] [Day 1 Universe]

### DISCUSSION THREAD

**Discussion by GeekTrainer** [community/community#74534] [Copilot Conversations]
## Coding with an AI pair programmer

**Speaker:** Christopher Harrison, Senior Developer Advocate
**Social:** You can follow me on [LinkedIn](https://linkedin.com/in/geektrainer)

**Time**: Wednesday 8th November 11:55 AM - 12:25 PM 

> The explosion of programming languages and frameworks has ironically made development more difficult. Developers need to remember different syntax across different environments, and are finding that a large percentage of code they write is tedious. Acting as an AI pair programmer, GitHub Copilot can help a developer offload those tasks and instead focus on the bigger picture. Let's explore how to get started with GitHub Copilot, its capabilities, and how to make the most out of the tool.

### Q&A time!

Thank you for joining my/our session/workshop! If you have any questions or feedback regarding what was presented, please feel free to add them in the comments and I'll/we'll do my best to answer. 

Once the recording is live I'll add a link.

**Comment by C34miller** [comment]
Really great demo with great comments on why developers should follow proper practices.

**Reply by ghostinhershell** [reply]
Agreed!

**Reply by mishmanners** [reply]
Nice work Chris!

**Comment by martijnmumbles** [comment]
I wanted to rewatch this today, but unfortunately it's been removed from the github universe portal?
Comes up as:
Video unavailable
This video has been removed by the uploader..

**Reply by gaurav-ketkar** [reply]
@martijnmumbles this issue has been reported internally. Someone will take a look at that soon!

**Reply by mishmanners** [reply]
@martijnmumbles please see the video posted on the main GitHub Universe page: https://reg.githubuniverse.com/flow/github/universe23/sessioncatalog/page/sessioncatalog/session/1694638932098001CYpy

**Reply by mishmanners** [reply]
All the sessions are being uploaded to the githubuniverse.com website 😄
